src/components/modals/DropOrEnrollModal.tsx

```diff
@@ -3,12 +3,28 @@ import { useForm } from "react-hook-form";
 import ModalLayout from "./ModalLayout";
 import InputField from "../ui/InputField";
 import Button from "../ui/Button";
-import { ApplicantDecision, ButtonVariant } from "../../utils/types";
+import { AlertType, ApplicantDecision, ButtonVariant } from "../../utils/types";
 import TextArea from "../ui/TextArea";
-import { useApplicantDecisionMutation } from "../../features/user/apiSlice";
+import { useApplicantDecisionMutation } from "../../features/user/backendApi";
 import Loader from "../ui/Loader";
-import Alert from "../ui/Alert";
 import useAutoCloseModal from "../../utils/hooks/useAutoCloseModal";
+import { getErrorInfo } from "../../utils/helper";
+import { handleShowAlert } from "../../utils/handleShowAlert";
+import { useDispatch } from "react-redux";
+
+// created dummy stages for testing purposes, will be removed soon when we get actual data
```

Comment:
I would that then move into a file like "stages.mock.ts" that you can also use in your tests.